serialization.md

```diff
@@ -0,0 +1,43 @@
+# CloudEvents Serialization Profile - Version 0.1
+
+This document specified how a CloudEvent is to be serialized into certain
```

Comment:
Should this be applied here? s/specified/specifies/

Reply:
fixed the "specified" typo. What did you mean by "should this applied here?" ?